### PR TITLE
Fix travis build process

### PR DESCRIPTION
**Pull request by prateekpandey14**
**What this PR does / why we need it**
This PR will fix the build issue in travis. Now travis will build mayactl as well as maya-apiserver
binaries and if this is tagged by github it will release and deploy them in Github release pages
 
fixes openebs/openebs#599
Partial fix for openebs/openebs#597

### CODE REVIEW

File: GNUmakefile [hunk 1]
```diff
@@ -23,7 +23,7 @@ APISERVER=maya-apiserver
 # Specify the date o build
 BUILD_DATE = $(shell date +'%Y%m%d%H%M%S')
 
-all: test
+all: test bin apiserver
```
Comment (AmitKumarDas) [inline GNUmakefile]: Can the target bin be renamed to mayactl or something more appropriate ?

File: buildscripts/build.sh [hunk 2]
```diff
@@ -28,15 +28,15 @@ XC_OSS=(${XC_OS// / })
 
 # Delete the old dir
 echo "==> Removing old directory..."
-rm -rf bin/*
-mkdir -p bin/
+rm -rf bin/maya/*
```
Comment (AmitKumarDas) [inline buildscripts/build.sh]: Will it be good to move this file to buildscripts/mayactl ?

File: GNUmakefile (hunk 1 above)
Comment (prateekpandey14) [reply GNUmakefile]: Naming it as `mayactl`

File: buildscripts/build.sh (hunk 2 above)
Comment (prateekpandey14) [reply buildscripts/build.sh]: sure !!!

**Comment by prateekpandey14** [comment]
Done !!

File: GNUmakefile [hunk 3]
```diff
@@ -23,16 +23,16 @@ APISERVER=maya-apiserver
...
-	@MAYACTL=${MAYACTL} MAYA_DEV=1 sh -c "'$(PWD)/buildscripts/build.sh'"
+	@MAYACTL=${MAYACTL} MAYA_DEV=1 sh -c "'$(PWD)/buildscripts/ctl/build.sh'"
 
-bin:
+mayactl:
 	@echo "----------------------------"
 	@echo "--> maya                    "
 	@echo "----------------------------"
-	@MAYACTL=${MAYACTL} sh -c "'$(PWD)/buildscripts/build.sh'"
+	@MAYACTL=${MAYACTL} sh -c "'$(PWD)/buildscripts/ctl/build.sh'"
```
Comment (AmitKumarDas) [inline GNUmakefile]: well why ctl ? & why not mayactl ?

**Pull request merged**